packages/sra-report/test/test_runner.ts

```diff
@@ -0,0 +1,138 @@
+                nockInterceptor.reply(200, {});
+                const summary = await utils.newmanRunAsync(newmanRunOptions);
+                const error = findAssertionErrorIfExists(
+                    summary,
+                    postmanCollectionRequestName,
+                    CONTENT_TYPE_ASSERTION_NAME,
+                );
+                const errorMessage = _.get(error, 'message');
+                expect(error).to.not.be.undefined();
+                expect(errorMessage).to.equal(`expected response to have header with key 'Content-Type'`);
+            });
+            it('fails when Content-Type header exists but not with value application/json', async () => {
+                const headers = {
+                    'Content-Type': 'text/html',
+                };
+                nockInterceptor.reply(200, {}, headers);
+                const summary = await utils.newmanRunAsync(newmanRunOptions);
+                const error = findAssertionErrorIfExists(
+                    summary,
+                    postmanCollectionRequestName,
+                    CONTENT_TYPE_ASSERTION_NAME,
+                );
+                const errorMessage = _.get(error, 'message');
+                expect(error).to.not.be.undefined();
+                expect(errorMessage).to.equal(`expected 'text/html' to include 'application/json'`);
+            });
+            it('passes when Content-Type header exists with value application/json', async () => {
+                const headers = {
+                    'Content-Type': 'charset=utf-8; application/json',
+                };
+                nockInterceptor.reply(200, {}, headers);
+                const summary = await utils.newmanRunAsync(newmanRunOptions);
+                const error = findAssertionErrorIfExists(
+                    summary,
+                    postmanCollectionRequestName,
+                    CONTENT_TYPE_ASSERTION_NAME,
+                );
+                expect(error).to.be.undefined();
+            });
+        });
+    },
+    runSchemaTests(
+        nockInterceptor: nock.Interceptor,
+        postmanCollectionFolderName: string,
+        postmanCollectionRequestName: string,
+        malformedJson: object,
+        correctJson: object,
+    ) {
+        const newmanRunOptions = {
+            ...baseNewmanRunOptions,
+            folder: postmanCollectionFolderName,
+        };
+        describe(SCHEMA_ASSERTION_NAME, () => {
+            it('fails when schema is invalid', async () => {
+                nockInterceptor.reply(200, malformedJson);
+                const summary = await utils.newmanRunAsync(newmanRunOptions);
+                const error = findAssertionErrorIfExists(summary, postmanCollectionRequestName, SCHEMA_ASSERTION_NAME);
+                const errorMessage = _.get(error, 'message');
+                expect(error).to.not.be.undefined();
+                expect(errorMessage).to.equal(`expected false to be true`);
```

Comment:
`expected false to be true` is a very unhelpful error message. Can we throw something better?

Reply:
Also can use regular string here.

Reply:
unfortunately i dont know if we can improve here, the way I wrote the assertion is the same way postman's docs suggest testing schema validation